<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_trem1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>T1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 255);</string>
    </property>
    <property name="text">
     <string>T2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(120, 110, 255);</string>
    </property>
    <property name="text">
     <string>T4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 26, 255);</string>
    </property>
    <property name="text">
     <string>T5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 96, 96);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>T3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>150</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_7">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="vel_trem1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem5_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>T1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem2_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>390</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 255);</string>
    </property>
    <property name="text">
     <string>T2</string>
    </property>
   </widget>
   <widget class="QSlider" name="vel_trem2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>390</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vel_trem3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>420</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem2_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>420</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 96, 96);</string>
    </property>
    <property name="text">
     <string>T3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem2_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>450</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(120, 110, 255);</string>
    </property>
    <property name="text">
     <string>T4</string>
    </property>
   </widget>
   <widget class="QSlider" name="vel_trem4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>450</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vel_trem5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>480</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem2_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>480</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 26, 255);</string>
    </property>
    <property name="text">
     <string>T5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>POSIÇÕES</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>876</x>
      <y>70</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>(x, y)</string>
    </property>
   </widget>
   <widget class="QLabel" name="t1x">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>100</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="t1y">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>100</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="t2y">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>120</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="t2x">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>120</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="t5y">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>210</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="t3x">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>150</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="t4y">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>180</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="t4x">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>180</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="t3y">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>150</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="t5x">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>210</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>320</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>VELOCIDADES</string>
    </property>
   </widget>
   <widget class="QLabel" name="t1Vel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>360</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="t2Vel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>390</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="t3Vel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>420</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="t4Vel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>450</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="t5Vel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>480</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>170</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>170</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <zorder>label_trilho2</zorder>
   <zorder>label_trilho2_2</zorder>
   <zorder>label_trilho4_2</zorder>
   <zorder>label_trilho3_2</zorder>
   <zorder>label_trilho4_3</zorder>
   <zorder>label_trilho2_7</zorder>
   <zorder>label_trilho2_4</zorder>
   <zorder>label_trilho2_3</zorder>
   <zorder>label_trilho1</zorder>
   <zorder>label_trilho3</zorder>
   <zorder>label_trilho4</zorder>
   <zorder>label_trilho1_2</zorder>
   <zorder>label_trilho4_4</zorder>
   <zorder>label_trilho2_5</zorder>
   <zorder>label_trilho3_3</zorder>
   <zorder>label_trilho2_6</zorder>
   <zorder>vel_trem1</zorder>
   <zorder>label_trem5_2</zorder>
   <zorder>label_trem2_2</zorder>
   <zorder>vel_trem2</zorder>
   <zorder>vel_trem3</zorder>
   <zorder>label_trem2_3</zorder>
   <zorder>label_trem2_4</zorder>
   <zorder>vel_trem4</zorder>
   <zorder>vel_trem5</zorder>
   <zorder>label_trem2_5</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>t1x</zorder>
   <zorder>t1y</zorder>
   <zorder>t2y</zorder>
   <zorder>t2x</zorder>
   <zorder>t5y</zorder>
   <zorder>t3x</zorder>
   <zorder>t4y</zorder>
   <zorder>t4x</zorder>
   <zorder>t3y</zorder>
   <zorder>t5x</zorder>
   <zorder>label_3</zorder>
   <zorder>t1Vel</zorder>
   <zorder>t2Vel</zorder>
   <zorder>t3Vel</zorder>
   <zorder>t4Vel</zorder>
   <zorder>t5Vel</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_trem4</zorder>
   <zorder>label_trem3</zorder>
   <zorder>label_trem1</zorder>
   <zorder>label_trem2</zorder>
   <zorder>label_trem5</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>962</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
